docker/Dockerfile

```diff
@@ -66,10 +70,12 @@ RUN set -ex; \
         unzip ${PLUGIN_PACKAGE_NAME}; \
         rm -f ${PLUGIN_PACKAGE_NAME}
 
-
 COPY docker-entrypoint.sh /usr/local/bin/
 
 RUN chmod +x /usr/local/bin/docker-entrypoint.sh
 
 ENTRYPOINT ["/usr/local/bin/docker-entrypoint.sh"]
 CMD ["apache2-foreground"]
+
+EXPOSE 80
```

Comment:
Exposing two ports?

Reply:
I am exposing port 80 for http, but can disable it.

Reply:
If it only works over https, then I'd just keep that one